keel-web/keel-web.gradle.kts

```diff
@@ -21,6 +21,7 @@ dependencies {
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
   implementation("com.netflix.spinnaker.kork:kork-core")
   implementation("com.netflix.spinnaker.kork:kork-web")
+  implementation("com.netflix.spinnaker.kork:kork-artifacts")
```

Comment:
I thought you were trying to remove the `kork-artifacts` dependency?

Reply:
Yes, this is only temporary while kork's `Artifact` coexists with the new `SpinnakerArtifact` in this PR.

Reply:
Ah, okay, got it.